src/DDTrace/Integrations/Eloquent/EloquentSandboxedIntegration.php

```diff
@@ -0,0 +1,105 @@
+use DDTrace\Integrations\SandboxedIntegration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+
+class EloquentSandboxedIntegration extends SandboxedIntegration
+{
+    const NAME = 'eloquent';
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * Add instrumentation to PDO requests
+     */
+    public function init()
+    {
+        $integration = $this;
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Builder',
+            'getModels',
+            function (SpanData $span, array $args) use ($integration) {
+                $span->name = 'eloquent.get';
+                $sql = $this->getQuery()->toSql();
+                $span->resource = $sql;
+                $span->meta[Tag::DB_STATEMENT] = $sql;
+                $integration->setCommonValues($span);
+            }
+        );
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Model',
+            'performInsert',
+            function (SpanData $span, array $args) use ($integration) {
+                $span->name = 'eloquent.insert';
+                $span->resource = get_class($this);
+                $integration->setCommonValues($span);
+            }
+        );
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Model',
+            'performUpdate',
+            function (SpanData $span, array $args) use ($integration) {
+                $span->name = 'eloquent.update';
+                $span->resource = get_class($this);
+                $integration->setCommonValues($span);
+            }
+        );
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Model',
+            'delete',
+            function (SpanData $span, array $args) use ($integration) {
+                $span->name = 'eloquent.delete';
+                $span->resource = get_class($this);
+                $integration->setCommonValues($span);
+            }
+        );
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Model',
+            'destroy',
+            function (SpanData $span) use ($integration) {
+                $span->name = 'eloquent.destroy';
+                $span->resource = get_called_class();
+                $integration->setCommonValues($span);
+            }
+        );
+
+        dd_trace_method(
+            'Illuminate\Database\Eloquent\Model',
+            'refresh',
+            function (SpanData $span) use ($integration) {
+                $span->name = 'eloquent.refresh';
+                $span->resource = get_class($this);
+                $integration->setCommonValues($span);
+            }
+        );
+
+        return Integration::LOADED;
+    }
+
+    /**
+     * Set common values shared by many different spans.
+     *
+     * @param SpanData $span
+     */
+    public function setCommonValues(SpanData $span)
+    {
+        $span->type = Type::SQL;
```

Comment:
We're missing the service name here. We'll need to add something similar to [this method](https://github.com/DataDog/dd-trace-php/blob/8ca38744500482fe9c2d6e642f227012f49e5fd1/src/DDTrace/Integrations/Laravel/V5/LaravelIntegrationLoader.php#L189-L200) as a static method and then add:

```php
$span->service = EloquentSandboxedIntegration::getAppName();
```

Or grab it in the constructor to reduce the number of method calls and then we could just do something like:

```php
$span->service = $this->serviceName;
```

Reply:
Great catch @SammyK